src/BookList.js

```diff
@@ -28,10 +29,11 @@ export default function BookList() {
   if (error) return <p>ERROR</p>;
   return (
     <List>
-      {data.getBooks.map(({ id, title, author }) => (
+      {data.getBooks.map(({ id, title, author, pages }) => (
         <ListItem key={id}>
           <Typography variant='body1' className={classes.title}>{title}</Typography>
-          <Typography variant='subtitle2' color='primary'>{author}</Typography>
+          <Typography variant='subtitle2' className={classes.title} color='primary'>{author}</Typography>
```

Comment:
Why are we reusing the `title` class?
If we're reusing the `title` class maybe let's change the name since we're using it for `author` too now.

Reply:
I remove the class and change the style to Grid.